Tiltfile

```diff
@@ -293,13 +293,60 @@ def enable_provider(name, debug):
             links = links,
         )
 
+        provider_yaml = get_provider(yaml)
+        print("provider_yaml: " + provider_yaml)
+        k8s_yaml(blob(provider_yaml))
+
 def find_manager(yaml):
     manifests = decode_yaml_stream(yaml)
     for m in manifests:
         if m["kind"] == "Deployment":
             return m["metadata"]["name"]
     return ""
 
+# This code reimplements cmd/clusterctl/client/repository/components.InventoryObject
+# It takes all required information from the Namespace resource.
+def get_provider(yaml):
+    manifests = decode_yaml_stream(yaml)
```

Comment:
We are doing this twice, one here, one on L301. Not sure how expensive this operation is, but it would be nice to dedup